content/uiux/concepts/crazy-eights/crazy-eights.md

```diff
@@ -0,0 +1,13 @@
+---
+Title: 'Crazy Eights'
+Description: 'A sketching exercise that challenges people to sketch eight distinct ideas in eight minutes'
+CatalogContent:
+  - 'intro-to-ui-ux'
+  - 'paths/front-end-engineer-career-path'
+---
+
+**Crazy Eights's** a Design Sprint method: meaning it is a quick sketching exercise that challenges people to sketch eight different ideas in eight minutes. The goal is to push beyond your first idea, which is usually basic(the least innovative), and to generate a wide variety of solutions to your initial design.
+
+#### Example  
+To properly implement the crazy eight method would be to 1st begin with an idea in mind for your design then setting a 8minute timer and folding a piece of paper 3 times so that you have eight boxes you then start the timer begin sketching and spend no more than 1 minute per box. Do this with other members in your team the more the ideas the better since this is just the drafting stage.
```

Comment:
```suggestion
The Crazy Eights methodology is used in a team setting to gather a multitude of ideas. To implement this method follow the steps below: 
1. Begin with a problem statement to solve for.
2. Take a sheet of paper and divide it into 8 sections.
3. Set a 8 minute timer.
4. Start the timer and begin sketching solutions for the problem at hand. 

> **Note:** Spend no more than 1 minute per idea. 

Once the activity is complete, the ideas generated by the entire team are used to finalize a solution.
```
I have structured this example to be steps for crazy eights as it makes more sense for this entry and removed other grammatical errors.

Reply:
done.